widgets/menu.js

```diff
@@ -0,0 +1,77 @@
+var React = require('react');
+var cx = require('classnames');
+
+var utils = require('../lib/widget-utils.js');
+
+var defaultTemplate = `<a href="{{href}}">{{name}}</a> {{count}}`;
```

Comment:
I would not use a `{{href}}` variable here, rather put `#` or remove the link, no?